Assets/Scripts/Map/Layout.cs

```diff
@@ -24,9 +24,9 @@ public class Layout : ScriptableObject
         [SerializeField] private int relaxIterations;
         [SerializeField] [Range(0f, 1f)] private float relaxStrength;
 
-        [field: SerializeField] private Graph<Vertex> VertexGraph { get; set; }
+        [field: SerializeField] public Graph<Vertex> VertexGraph { get; set; }
```

Comment:
These are meant to be private, I'd prefer for the architecture they remain that way and you use accessors to get the values

Reply:
I need access to them above. It seemed silly to write a forwarding method in-between since I'm going to need them as a graph anyway.